spyderlib/plugins/configdialog.py

```diff
@@ -903,77 +917,419 @@ def _save_lang(self):
-                                            name, option, without_layout=True)
-                    label.setAlignment(Qt.AlignRight|Qt.AlignVCenter)
-                    cs_layout.addWidget(label, row+1, 0)
-                    cs_layout.addLayout(clayout, row+1, 1)
-                    cs_layout.addWidget(cb_bold, row+1, 2)
-                    cs_layout.addWidget(cb_italic, row+1, 3)
-            cs_group.setLayout(cs_layout)
-            if tabname in sh.COLOR_SCHEME_NAMES:
-                def_btn = self.create_button(_("Reset to default values"),
-                                         lambda: self.reset_to_default(tabname))
-                tabs.addTab(self.create_tab(cs_group, def_btn), tabname)
-            else:
-                tabs.addTab(self.create_tab(cs_group), tabname)
-        
+        names = [n for n in self.get_option("names")]
+        custom_names = self.get_option("custom_names")
+
+        # Widgets
+        about_label = QLabel(_("The existing schemes can be edited or you "
+                               "can create new custom color schemes.<br>"))
+        edit_button = QPushButton(_("Edit selected"))
+        create_button = QPushButton(_("Create new scheme"))
+        self.delete_button = QPushButton(_("Delete"))
+        self.preview_editor = CodeEditor(self)
+        self.stacked_widget = QStackedWidget(self)
+        self.reset_button = QPushButton(_("Reset"))
+        self.scheme_editor_dialog = SchemeEditor(parent=self,
+                                                 stack=self.stacked_widget)
+
+        # Widget setup
+        about_label.setWordWrap(True)
+        schemes_combo = self.create_combobox(_('Scheme:'),
+                                             [('', '')],
+                                             'selected')
+        self.schemes_combo = schemes_combo.combobox
+
+        # Layouts
         vlayout = QVBoxLayout()
-        vlayout.addWidget(tabs)
+
+        combo_layout = QHBoxLayout()
+        combo_layout.addWidget(schemes_combo.label)
+        combo_layout.addWidget(schemes_combo.combobox)
+
+        buttons_layout = QVBoxLayout()
+        buttons_layout.addLayout(combo_layout)
+        buttons_layout.addWidget(edit_button)
+        buttons_layout.addWidget(self.reset_button)
+        buttons_layout.addWidget(self.delete_button)
+        buttons_layout.addStretch(1)
+        buttons_layout.addWidget(create_button)
+
+        preview_layout = QVBoxLayout()
+        preview_layout.addWidget(self.preview_editor)
+
+        buttons_preview_layout = QHBoxLayout()
+        buttons_preview_layout.addLayout(buttons_layout)
+        buttons_preview_layout.addLayout(preview_layout)
+
+        manage_layout = QVBoxLayout()
+        manage_layout.addWidget(about_label)
+        manage_layout.addLayout(buttons_preview_layout)
+        manage_group = QGroupBox(_("Manage color schemes"))
+        manage_group.setLayout(manage_layout)
+
+        vlayout.addWidget(manage_group)

```

Comment:
Could you move these lines (from `manage_layout = QVBoxLayout()` up to here) before `combo_layout = QHBoxLayout()`?

I think this way it'd be easier to read this code because the order it appears here corresponds to how it's shown visually.